unumpy/tests/test_numpy.py

```diff
@@ -318,6 +319,6 @@ def test_ufuncs_results(backend, method, args, kwargs, res):
         pytest.xfail(reason="The backend has no implementation for this ufunc.")
 
     if isinstance(ret, sparse.SparseArray):
```

Comment:
Don't need this line or the one below.

Reply:
The `sparse.todense` method seems necessary at least for the `ret` variable, otherwise `np.equal` fails.